docker_master.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
```

Comment:
add if running or hardcode command 'docker-compose down', i had it already running and bad things happened

Reply:
added